packages/http/src/mocker/generator/JSONSchema.ts

```diff
@@ -1,3 +1,5 @@
+// @ts-ignore
```

Comment:
This is not required if you install `@types/faker` in the repository root.

`yarn add -D -W @types/faker`

Reply:
395fd7af5f3ff9305bf89b9969b31debd2771c29